### PR TITLE
Added tests for `Board` `toString` method

### PR DESCRIPTION
**Pull request by peiva-git**

### CODE REVIEW

**Comment by peiva-git** [comment]
Checks are failing because the branvh was created before the `getAdjacentPositions` method bugfix

**Comment by manuelkoso** [comment]
You can merge this branch with the branch where you fixed the bug in order to pass the tests

File: src/main/java/Board.java [hunk 1]
```diff
@@ -133,14 +137,17 @@ public String toString() {
...
+        sb.append(" ");
+        for (int j = 0; j < numberOfColumns; j++) {
+            sb.append("  ").append((char) ('A' + j));
+        }
         return sb.toString();
     }
 
     @NotNull
     @Override
-    public Iterator<Map.Entry<Position, Cell>> iterator() {
```
Comment (manuelkoso) [inline src/main/java/Board.java:142]: We need also the color of the stone when iterating

**Review by manuelkoso — COMMENTED** [review]
Check the comment

**Comment by manuelkoso** [comment]
Why did you merge into the master branch?

**Comment by peiva-git** [comment]
Ops my bad

File: src/main/java/Board.java (hunk 1 above)
Comment (peiva-git) [reply src/main/java/Board.java:142]: The color of the stone can be obtained from the position if needed

**Pull request merged**